cli/crates/server/src/bridge/listener/server.rs

```diff
@@ -71,16 +72,34 @@ async fn event_listener(worker_port: u16) -> Result<(), ServerError> {
                 })
                 .collect::<Vec<_>>();
 
-            client
-                .post(format!("http://127.0.0.1:{worker_port}/stream-router/main/dynamodb"))
+            let response = client
+                .post(format!(
+                    "http://127.0.0.1:{worker_port}/stream-router/main/dynamodb/{DEFAULT_AWS_REGION}"
+                ))
                 .header("x-api-key", CLI_API_KEY)
                 .json(&dynamo_events)
                 .send()
                 .await
                 // TODO: consider if this should panic or show a specific error to the user
                 .expect("could not contact the stream router");
-
-            trace!("sent event to stream-router");
+            if response.status() != 200 {
+                trace!(
+                    "Failed to send event to stream-router. Responded: {:?}\n {}",
+                    response.headers().clone(),
+                    match response.text().await {
+                        Ok(text) => text,
+                        Err(err) => err.to_string(),
+                    }
+                );
+            } else {
+                trace!(
+                    "Successfully sent event to stream-router. Responded: {}",
```

Comment:
I'd maybe split this into a trace before sending the event and a trace with the response rather than differentiating based on the code.

Since we don't need the response itself we should consider whether we want to await it for tracing purposes

Reply:
@Finistere